vendor/github.com/livepeer/go-livepeer-basicnet/network_node.go

```diff
@@ -74,7 +74,7 @@ func NewNode(listenPort int, priv crypto.PrivKey, pub crypto.PubKey, f *BasicNot
 	}
 	rHost := rhost.Wrap(basicHost, dht)
 
-	glog.Infof("Created node: %v at %v", peer.IDHexEncode(rHost.ID()), rHost.Addrs())
+	glog.V(2).Infof("Created node: %v at %v", peer.IDHexEncode(rHost.ID()), rHost.Addrs())
```

Comment:
This is mainly for my own knowledge - is the choice of V level for log statements just based on the need to group certain log statements together and it doesn't matter what V level you actually use? Or does each V level actually correspond to some severity level i.e. 2 -> warnings

Reply:
Good point.  I'll document the levels.